ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/ExpenseItemDetailView.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Activity that lets the user view 
+ * information of the selected ExpenseItem
+ * 
+ * Covers Issue 16
+ * @author Oleg Oleynikov 
+ * @version 1.0
+ * @since 2015-03-15
+ */
+
+package ca.ualberta.cs.shinyexpensetracker;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Locale;
+
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaim;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaimList;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaimsRepository;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseItem;
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.TextView;
+
+public class ExpenseItemDetailView extends Activity {
+
+	private ExpenseItem item;
+	private Context context = ExpenseItemDetailView.this;
+	
+	@Override
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setContentView(R.layout.activity_expense_item_detail_view);
+		
+		Intent intent = getIntent();
+		Bundle bundle = intent.getExtras();
+		
+		if (bundle != null){
+			int claimId = (Integer) bundle.get("Claim ID");
+			int expenseItemId = (Integer) bundle.get("Item ID");
+			ExpenseClaimController controller = ExpenseClaimController.getInstance();
+			ExpenseClaim claim = controller.getExpenseClaim(claimId);
+			item = claim.getItemById(expenseItemId);
+			populateTextViews();
+		}
+		// this one is just for testing purposes
+		// because I can't figure out how to send an Intent from a test
+		/*else{
+			ExpenseClaimController controller = ExpenseClaimController.getInstance();
+			ExpenseClaim claim = controller.getExpenseClaim(0);
+			item = claim.getItemById(0);
+			populateTextViews();
+		}*/
+		
+	}
+
+	private void populateTextViews(){
+		SimpleDateFormat dateFormatter = new SimpleDateFormat("MM-dd-yyyy", Locale.CANADA);
+		
+		TextView name = (TextView) findViewById(R.id.expenseItemNameValue);
+		TextView date = (TextView) findViewById(R.id.expenseItemDateValue);
+		TextView category = (TextView) findViewById(R.id.expenseItemCategoryValue);
+		TextView description = (TextView) findViewById(R.id.expenseItemDescriptionValue);
+		TextView amount = (TextView) findViewById(R.id.expenseItemAmountValue);
+		TextView currency = (TextView) findViewById(R.id.expenseItemCurrencyValue);
+		TextView receipt = (TextView) findViewById(R.id.expenseItemReceiptValue);
+		TextView completeness = (TextView) findViewById(R.id.expenseItemCompletenessValue);
```

Comment:
I just have no idea what completeness is supposed to mean. In the use case (#16)

* The system displays the date the expense was incurred, the category, the textual description, amount spent, unit of currency, whether there is a photographic receipt, and **incompleteness indicator**.

That is supposed to be an ExpenseItem feature, but I don't even know what it could possibly mean